recipes/plumbum/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "plumbum" %}
+{% set version = "1.6.2" %}
+{% set sha256 = "75eff3a55e056d8fc06f7b7ceb603ce4c26650cd6a2196bcdb0b80fee59471a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - paramiko
+
+test:
+  imports:
+    - plumbum
+    - plumbum.cli
+    - plumbum.colorlib
+    - plumbum.commands
+    - plumbum.fs
+    - plumbum.machines
+    - plumbum.path
+
+  requires:
```

Comment:
Unless you are running tests, the simple import checks shouldn't require either of these modules. `paramiko` will be pulled in by the `run` requirement.